flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/operators/co/AsyncIntervalJoinOperator.java

```diff
@@ -417,12 +419,12 @@ public <X> void output(OutputTag<X> outputTag, X value) {
     }
 
     @VisibleForTesting
-    MapState<Long, List<IntervalJoinOperator.BufferEntry<T1>>> getLeftBuffer() {
+    public MapState<Long, List<IntervalJoinOperator.BufferEntry<T1>>> getLeftBuffer() {
```

Comment:
Seems unnecessary?

Reply:
Because `AsyncIntervalJoinOperator` is moved into `asyncprocessing` package, and `AsyncIntervalJoinOperatorTest` still needs these methods.